plugins/module_utils/aci.py

```diff
@@ -1527,12 +1533,16 @@ def post_config(self):
             return
         elif not self.module.check_mode:
             # Sign and encode request as to APIC's wishes
+            url = self.url
+            if parent_class is not None:
+                url = "{protocol}://{host}/{path}".format(path=self.parent_path, **self.module.params)
```

Comment:
Let's confirm from @lhercot if this change is necessary -

```
if parent_class is not None:
    if self.params.get("port") is not None:
        url = "{protocol}://{host}:{port}/{path}".format(path=self.parent_path, **self.module.params)
    else:
        url = "{protocol}://{host}/{path}".format(path=self.parent_path, **self.module.params)
```

Reply:
Yes that is probably a good catch